libs/cart/guides/getting-started.md

```diff
@@ -0,0 +1,30 @@
+# Getting Started
+
+This overview assumes that you have already set up an Angular project and have gone through the [cart installation guide](./installation.md). If you have not, we recommend you do that first.
+
+## Setting up your `AppModule`
+
+To get started, import the `DaffCartModule` in your `AppModule`. Next, import `StoreModule.forRoot({})`, which will be relevant later on when utilizing the redux and state management features of the cart module.
+
+```typescript
+@ngModule({
+  imports:[
+    StoreModule.forRoot({}),
+    DaffCartModule
+  ]
+})
+export class AppModule {}
+```
+
+## Utilizing inside your component
```

Comment:
Can we use the word "use" instead of "utilize"? Utilize means to use for something other than its intended purpose, so it's almost always correct to use "use" instead of "utilize"

Reply:
changed

Reply:
changes